File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/game/SetOtherGameScore.java
package io.github.oleksivio.telegram.bot.api.model.method.game;

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.method.game.SetGameScore;
import io.github.oleksivio.telegram.bot.core.model.responses.BooleanResponse;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;

public class SetOtherGameScore extends SetGameScore<Boolean> {
    public SetOtherGameScore(ActionNetworker networker) {
        super(networker);
    }

    @Override
    protected Class<? extends CommonResponse<Boolean>> getResultWrapperClass() {
        return BooleanResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/game/SetOtherGameScore.kt
package io.github.oleksivio.telegram.bot.api.model.method.game

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.BooleanResponse
import io.github.oleksivio.telegram.bot.core.model.method.game.SetGameScore

class SetOtherGameScore(networker: ActionNetworker) : SetGameScore<Boolean>(networker) {
    override val resultWrapperClass = BooleanResponse::class
}
